HCCGo/main.js

```diff
@@ -29,5 +29,6 @@ app.on('ready', function() {
 
     mainWindow.once('ready-to-show', () => {
         mainWindow.show();
+        //mainWindow.webContents.openDevTools();
```

Comment:
Eh, having it in a "help" menu or something would be ok with me.

Anyways, I'll open an issue so we can discuss this further.  No need to hold up this pull request.